vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridItemRefreshPageIT.java

```diff
@@ -71,7 +71,7 @@ public void dataProviderRefreshItem_gridDoesntTouchOtherItemsOnDOM() {
         GridTHTDElement firstRowCell = grid.getCell(0, 1);
 
         Assert.assertEquals("Invalid cell content", "0", firstRowCell.getText());
-        Assert.assertEquals("Invalid cell content", "5", grid.getCell(5,1).getText());
+        Assert.assertEquals("Invalid cell con0tent", "5", grid.getCell(5,1).getText());
```

Comment:
Why was this changed?

Reply:
I do not know why it added 0 within content word, I will correct it back

Reply:
done